components/cd-article-list/cd-article-list.css

```diff
@@ -0,0 +1,68 @@
+.cd-article-list {
+  margin-top: 4.5rem;
+}
+
+.cd-article-list__title {
+  margin: 0 0 1.5rem 0;
+  color: #1f1f1f;
+  font-size: 1.3125rem;
+  font-weight: 700;
+  line-height: 1.5rem;
+}
+
+.cd-article-list__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.cd-article-list__list__item {
+  margin: 0;
+  padding: 0;
+  border: 1px solid #d8d8d8;
+  border-width: 1px 0 1px 0;
+}
+.cd-article-list__list__item + .cd-article-list__list__item {
+  border-width: 0 0 1px 0;
+}
+
+.cd-article-list-link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 1.25rem 0;
+  color: #1f1f1f;
+  text-decoration: none;
+}
+.cd-article-list-link:hover,
+.cd-article-list-link:focus,
+.cd-article-list-link:active  {
+  text-decoration: underline;
+}
+
+.cd-article-list-link__title {
+  flex: 1 1 auto;
+  font-size: 1rem;
+  line-height: 1.75rem;
+}
+@media (min-width: 768px) {
+  .cd-article-list-link__title {
+    font-size: 1.125rem;
+    line-height: 2rem;
+  }
+}
+
+.cd-article-list-link__arrow.cd-icon {
+  display: block;
+  flex: 0 0 auto;
+  width: 0.625rem;
```

Comment:
Since we are using REM units (based on 16px), I think it makes sense to try and keep the increments easy to mentally translate to 16 so instead of 0.625rem (which btw is 10px) I would use 0.5 or .75 (8 or 12px). This also ensures we stick to a grid-like system and will result in better visual consistency, and then we don't have random 10px elements amongst mainly 4, 8, 12, 16px elements. This is most important when it comes to margins and padding (spacing) but I think it can also apply to dimensions.